submissions/Jemikalajah-Philip/Html/homepage.html

```diff
@@ -57,7 +57,7 @@
                         <div class="col-md-3 col-sm-6">
                             <div class="product-grid">
                                 <div class="product-img">
-                                    <a href="#"><img class="img-7" src="icon/img7.jpeg" width="400px" height="300px"></a>
+                                    <a href="#"><img class="product" src="icon/img7.jpeg" width="400px" height="300px"></a>
```

Comment:
I'll advice you use a better class name. i.e `product--image` is a better class name to `product`. for the image element.